src/main/scala/scala/meta/fastpass/internal/FastpassEnrichments.scala

```diff
@@ -0,0 +1,114 @@
+package scala.meta.fastpass.internal
```

Comment:
Let's rename this package
```suggestion
package scala.meta.internal.fastpass
```

Everything should be in internal except the main method, which can live in `scala.meta.fastpass.Fastpass`.

Reply:
I've moved everything to `scala.meta.internal.fastpass`.

The current main is still `BloopPants`. I've left it in `scala.meta.internal.fastpass.pantsbuild`, because it's still specific to Pants, and I didn't want to refactor things before adding support for other tools. It'll most likely be removed then.